scripts/app.js

```diff
@@ -0,0 +1,76 @@
+var projects = [];
+
+function Projects (projectObject) {
+  for(var prop in projectObject) {this[prop]= projectObject[prop];}
+
+}
+
+ timeSinceCreated = function(createdDate){
+ 	myDays = parseInt((new Date() - new Date(createdDate))/60/60/24/1000);
+
+ 	if (myDays < 365) {
+	return myDays + ' days ago'
+ 		
+ 	} else return parseInt(myDays / 365) + ' years ago';
+
+ }
+Projects.prototype.toHtml = function() {
+  var $tempProject = $('article.template').clone();
+
+  $tempProject.attr('data-category', this.category);
+  $tempProject.find('h5').html(this.name);
+  console.log(this.name);
+  $tempProject.find('address a').html(this.maker);
+
+
+  switch (this.mediaType ){
```

Comment:
Is there a way to simplify this and avoid the bulky switch statement? (I don't actually know, but it might be worth thinking about)